tutorials/misc/gles2_gles3_differences.rst

```diff
@@ -195,3 +195,20 @@ For a complete list of built-in GLSL functions see the :ref:`Shading Language do
 +------------------------------------------------------------------------+--------------------------------------------------+
 | vec_type **fwidth** ( vec_type )                                       |                                                  |
 +------------------------------------------------------------------------+--------------------------------------------------+
+
+Godot also provides many built-in variables and render modes. Some cannot be supported in GLES2. Below is a list of
+built-in variables and render modes that, when written to, will have no effect when using the GLES2 backend.
```

Comment:
I'd prefer something like "will have no effect or could even cause issues".

Reply:
Thanks! Should be good now.